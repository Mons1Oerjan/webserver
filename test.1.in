8080
0.0 0.2 sws.c
0.0 1.0 network.c
0.0 0.4 network.h
0.0 1.0 makefile
0.0 1.2 README.md
0.0 1.4 sws.c
0.0 0.1 network.c
0.0 5.0 network.h
0.0 0.3 makefile
0.0 1.1 hydra.py
0.0 2.0 README.md
