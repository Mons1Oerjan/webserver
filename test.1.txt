//	Test Case 1 Description
//	CSCI 3120 Winter 2017
//	Group 16

Purpose of test: Solely testing the delays function, every request comes in at
                 the same time.

How it works: Takes each request at the same time and then the connections are
              delayed by different amounts of times.

Expected Results: 7, 1, 9, 3, 2, 4, 10, 5, 6, 11, 8
