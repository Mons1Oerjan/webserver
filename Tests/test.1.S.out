Thread 1 ============================================
HTTP/1.1 400 Bad Request
Content-Length: 0
Connection: close
Server: Jetty(9.2.3.v20140905)


Thread 2 ============================================
HTTP/1.1 400 Bad Request
Content-Length: 0
Connection: close
Server: Jetty(9.2.3.v20140905)


Thread 3 ============================================
HTTP/1.1 400 Bad Request
Content-Length: 0
Connection: close
Server: Jetty(9.2.3.v20140905)


Thread 4 ============================================
HTTP/1.1 400 Bad Request
Content-Length: 0
Connection: close
Server: Jetty(9.2.3.v20140905)


Thread 5 ============================================
HTTP/1.1 400 Bad Request
Content-Length: 0
Connection: close
Server: Jetty(9.2.3.v20140905)


Thread 6 ============================================
HTTP/1.1 400 Bad Request
Content-Length: 0
Connection: close
Server: Jetty(9.2.3.v20140905)


Thread 7 ============================================
HTTP/1.1 400 Bad Request
Content-Length: 0
Connection: close
Server: Jetty(9.2.3.v20140905)


Thread 8 ============================================
HTTP/1.1 400 Bad Request
Content-Length: 0
Connection: close
Server: Jetty(9.2.3.v20140905)


Thread 9 ============================================
HTTP/1.1 400 Bad Request
Content-Length: 0
Connection: close
Server: Jetty(9.2.3.v20140905)


Thread 10 ============================================
HTTP/1.1 400 Bad Request
Content-Length: 0
Connection: close
Server: Jetty(9.2.3.v20140905)


Thread 11 ============================================
HTTP/1.1 400 Bad Request
Content-Length: 0
Connection: close
Server: Jetty(9.2.3.v20140905)


=====================================================================
Request completion order:  7  1  9  3  2  4  10  5  6  11  8 
